extensions/NV/NV_shader_atomic_int64.txt

```diff
@@ -183,6 +183,11 @@ Issues
 
 Revision History
 
+    Revision 3, May 11, 2021
+
+      Rename assembly option from NV_shader_atomic_uint64 to
```

Comment:
I'd suggest:

> Fix typos in the NV_gpu_program5 assembly option name, which should have been specified as "NV_shader_atomic_int64".

to clarify that you're fixing a bug in the spec rather than making a functional change.  Otherwise, looks great - thanks.